llama_stack/providers/registry/inference.py

```diff
@@ -98,9 +98,7 @@ def available_providers() -> List[ProviderSpec]:
             api=Api.inference,
             adapter=AdapterSpec(
                 adapter_id="bedrock",
-                pip_packages=[
-                    "boto3"
-                ],
+                pip_packages=["boto3"],
```

Comment:
It looks like you have a couple of Python changes (this one and another in augment_messages.py) that are unrelated to markdown linting.

Reply:
Yeah I think it just feels like they ran pre-commit and ended up doing formatting. Seems OK.

Reply:
Ah, makes sense.